src/routes/resetPasswordRoutes.js

```diff
@@ -1,8 +1,11 @@
 import { Router } from "express";
 import resetPasswordController from "../controllers/resetPasswordController";
+import Authentication from "../middlewares/authenticate";
+
+const { verifyUserByDetails, verifyUserById, verifyToken } = Authentication;
 
 const router = Router();
-router.post("/users/recover", resetPasswordController.recover);
-router.post("/users/reset/:id/:token", resetPasswordController.reset);
+router.post("/users/recover", verifyUserByDetails, resetPasswordController.recover);
```

Comment:
why are these changed to verifyUserbyID ????? what happened to what was there before?